examples/mnist/tf_example.py

```diff
@@ -0,0 +1,137 @@
+            test_batch_image, test_batch_label = tf.train.batch(
+                [test_image, test_label], batch_size=batch_size
+            )
+
+            # Train
+            print('Training model for {0} epochs with batch size {1} and log interval {2}'.format(
+                epochs, batch_size, log_interval
+            ))
+
+            with tf.Session() as sess:
+                sess.run([
+                    tf.local_variables_initializer(),
+                    tf.global_variables_initializer(),
+                ])
+                coord = tf.train.Coordinator()
+                threads = tf.train.start_queue_runners(sess=sess, coord=coord)
+                try:
+                    for i in range(epochs):
+                        if coord.should_stop():
+                            break
+
+                        feed_image_batch, feed_label_batch = sess.run([train_batch_image, train_batch_label])
+                        sess.run(train_step, feed_dict={x: feed_image_batch, y_: feed_label_batch})
+
+                        if (i % log_interval) == 0 or i == (epochs - 1):
+                            correct_prediction = tf.equal(tf.argmax(y, 1), y_)
+                            accuracy = tf.reduce_mean(tf.cast(correct_prediction, tf.float32))
+
+                            feed_image_batch, feed_label_batch = sess.run([test_batch_image, test_batch_label])
+                            print('After training for {0} epochs, the accuracy of the model is: {1:.2f}'.format(
+                                i,
+                                sess.run(accuracy, feed_dict={x: feed_image_batch, y_: feed_label_batch})))
+                finally:
+                    coord.request_stop()
+                    coord.join(threads)
+
+def transform(readout_example):
+    image = tf.cast(tf.reshape(readout_example.image, [784]), tf.float32)
+    label = readout_example.digit
+    return image, label
+
+
+def main():
+    # Training settings
+    parser = argparse.ArgumentParser(description='Petastorm Tensorflow MNIST Example')
+    default_dataset_url = 'file://{}'.format(DEFAULT_MNIST_DATA_PATH)
+    parser.add_argument('--dataset-url', type=str,
+                        default=default_dataset_url, metavar='S',
+                        help='hdfs:// or file:/// URL to the MNIST petastorm dataset'
+                             '(default: %s)' % default_dataset_url)
+    parser.add_argument('--epochs', type=int, default=100, metavar='N',
+                        help='number of epochs to train (default: 100)')
+    parser.add_argument('--batch-size', type=int, default=100, metavar='N',
+                        help='input batch size for training (default: 100)')
+    parser.add_argument('--log-interval', type=int, default=10, metavar='N',
+                        help='how many batches to wait before logging training status (default: 10)')
+    args = parser.parse_args()
+
+    train_and_test(
```

Comment:
Can you also please cover this with a test (same as what was done for pytorch). This will make sure the code is not getting stale.